### ISSUE TITLE
Request: Please fix camera zoom when moving the worm in 3rd person

### ISSUE THREAD

**Issue by jonesjb** [heatray/WUMPatch#8]
Can you please fix the zoom in and zoom out when navigating the worm in 3rd person view. Often we find that the camera is too close to the worm. This used to work in Worms 4 Mayhem, but no longer seems to function in Worms Ultimate Mayhem (despite control settings seeming to indicate otherwise). Would this be possible?

@heatray thank you for all your support on trying to make Worms Ultimate Mayhem an even better game, even after all these years.

**Comment by user-1-2-3-4-5** [comment]
I agree, the leading "unofficial patch" could fix the non-functional camera control option.

There could also be a way to fine-tune the camera distance, similar to scrolling (the mouse wheel by the default) in the aerial view (and additionally supporting changing the default field of view, which should also apply to the first-person view).
